activation/activation.go

```diff
@@ -211,6 +212,7 @@ func (b *Builder) StartSmeshing(coinbase types.Address, opts PostSetupOpts) erro
 			return nil
 		case err != nil:
 			b.log.Panic("initialization failed: %v", err)
+			return err
```

Comment:
Note: this is only needed to pass the unit test that mocks `log.Panic`... (and doesn't actually panic, allowing the code to proceed)